src/components/themed/QrCode.tsx

```diff
@@ -53,9 +52,29 @@ export function QrCode(props: Props) {
   const sizeInCells = code.getModuleCount() + 2 * cellsPadding
   const viewBox = `0 0 ${sizeInCells} ${sizeInCells}`
 
+  // Calculate crypto icon size and its parent background container
+  const iconContainerSize = size * 0.2
+  const iconContainerStyle = React.useMemo<ViewStyle>(
+    () => ({
+      position: 'absolute',
+      width: iconContainerSize,
+      height: iconContainerSize,
+      borderRadius: theme.cardBorderRadius,
+      backgroundColor: theme.qrBackgroundColor,
+      alignItems: 'center',
+      justifyContent: 'center',
+      left: '50%',
+      top: '50%',
+      transform: [{ translateX: -iconContainerSize / 2 }, { translateY: -iconContainerSize / 2 }]
```

Comment:
Optional: The transform may not be the most efficient option. You could also do it this way:

```ts
marginLeft: -iconContainerSize / 2,
marginTop: -iconContainerSize / 2
```

This will move the container up & to the left by half it's width, allowing `left: '50%'` and `top: '50%'` to correctly center it. It's very similar to the `transform` you have, but it happens at the layout stage instead of the rendering phase, which is generally perferable.